wolfssl/internal.h

```diff
@@ -2435,8 +2435,8 @@ typedef struct CipherSuite {
 #endif
 } CipherSuite;
 
-/* use wolfSSL_API visibility to be able to test in tests/api.c */
-WOLFSSL_API void InitSuitesHashSigAlgo(byte* hashSigAlgo, int have,
+/* use wolfSSL_TEST_API visibility to be able to test in tests/api.c */
```

Comment:
NIT: maybe this comment can be removed now, or moved in a more general position

Reply:
yep good thinking -- snipped them all out.